### PR TITLE
[VEN-2384] Redeem VToken and send funds to Binance 

### PR DESCRIPTION
**Pull request by narayanprusty**
## Description

Resolves VEN-2384

### CODE REVIEW

File: vips/vip-256/bscmainnet.ts [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+  {
+    symbol: "vXRP",
+    amount: parseUnits("534654.72490775", 8).toString(),
+    address: "0xB248a295732e0225acd3337607cc01068e3b9c10",
+  },
+  // {
+  //     symbol: "vTUSDOLD",
+  //     amount: parseUnits("234046.29380374", 8).toString(),
+  //     address: "0x08CEB3F4a7ed3500cA0982bcd0FC7816688084c3"
+  // },
```
[the remark this answers is not in the document]
Comment (chechu) [reply vips/vip-256/bscmainnet.ts:100]: There isn't enough liquidity in the market to redeem the VTokens

File: vips/vip-256/bscmainnet.ts [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    symbol: "vSXP",
+    amount: parseUnits("925.79433296", 8).toString(),
+    address: "0x2fF3d0F6990a40261c66E1ff2017aCBc282EB6d0",
+  },
+];
+
+export const BEP20Transfers = [
+  {
+    symbol: "BTCB",
+    amount: BigNumber.from(parseUnits("3.5029", 18)).add(parseUnits("1.06876", 18)).toString(),
```
Comment (kkirka) [inline vips/vip-256/bscmainnet.ts:161]: parseUnits should return BigNumber anyway
```suggestion
    amount: parseUnits("3.5029", 18).add(parseUnits("1.06876", 18)).toString(),
```

File: vips/vip-256/bscmainnet.ts [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+    signature: "withdrawTreasuryBEP20(address,uint256,address)",
+    params: [vTokenTransfer.address, vTokenTransfer.amount, TOKEN_REDEEMER],
+  };
+});
+
+const vBEP20RedeemCommands = vToken_Transfers.map((vTokenTransfer: VTokenTransfer) => {
+  return {
+    target: TOKEN_REDEEMER,
+    signature: "redeemAndTransfer(address,address)",
+    params: [vTokenTransfer.address, TREASURY],
```
Comment (kkirka) [inline vips/vip-256/bscmainnet.ts:273]: This way treasury would have some leftover tokens. Maybe we could transfer them directly to Binance? If not, we could just redeem without TOKEN_REDEEMER contract 🤔

**Review by kkirka — APPROVED** [review]

File: vips/vip-256/bscmainnet.ts [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+  {
+    symbol: "vUNI",
+    amount: parseUnits("2497.74135186", 8).toString(),
+    address: "0x27FF564707786720C71A2e5c1490A63266683612",
+  },
+  {
+    symbol: "vFDUSD",
+    amount: parseUnits("10355.09068368", 8).toString(),
+    address: "0xC4eF4229FEc74Ccfe17B2bdeF7715fAC740BA0ba",
+  },
```
Comment (chechu) [inline vips/vip-256/bscmainnet.ts:75]: Let's review if we should try to redeem vFDUSD. The launchpool will decrease the liquidity in the market

**Comment by kkirka** [comment]
I reviewed the commands and checked the numbers, the amounts correspond to the requirements specified in VEN-2384

File: vips/vip-256/bscmainnet.ts [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+    address: "0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82",
+  },
+  {
+    symbol: "UNI",
+    amount: parseUnits("1252.25385", 18).toString(),
+    address: "0xBf5140A22578168FD562DCcF235E5D43A02ce9B1",
+  },
+  {
+    symbol: "FDUSD",
+    amount: BigNumber.from(parseUnits("5261.65425", 18)).add(parseUnits("1031.64844", 18)).toString(),
```
Comment (chechu) [inline vips/vip-256/bscmainnet.ts:196]: If we don't redeem the vFDUSD tokens, we should transfer only `1031.64844`

File: vips/vip-256/bscmainnet.ts (hunk 2 above)
Comment (chechu) [reply vips/vip-256/bscmainnet.ts]: https://github.com/VenusProtocol/vips/pull/200/commits/8a2d7984d74e0aadb8ddcec54eb3fe6a5a47cd7c

**Comment by chechu** [comment]
https://app.venus.io/#/governance/proposal/253?chainId=56

**Pull request merged**